datafusion/core/tests/sql/aggregates.rs

```diff
@@ -1834,11 +1834,11 @@ async fn aggregate_avg_add() -> Result<()> {
     assert_eq!(results.len(), 1);
 
     let expected = vec![
-        "+--------------+-------------------------+-------------------------+-------------------------+",
-        "| AVG(test.c1) | AVG(test.c1) + Int64(1) | AVG(test.c1) + Int64(2) | Int64(1) + AVG(test.c1) |",
-        "+--------------+-------------------------+-------------------------+-------------------------+",
-        "| 1.5          | 2.5                     | 3.5                     | 2.5                     |",
-        "+--------------+-------------------------+-------------------------+-------------------------+",
+        "+--------------+---------------------------+---------------------------+---------------------------+",
```

Comment:
I just have the comments about the header of the expr.
The input sql is  `AGG(C1) + 1`, `1` is the int64 data type, but the header is convert to float after casted

Reply:
Do we have the method to make the header consistent, and it can be changed with the changes of the optimizer plan.
cc @andygrove  @alamb

Reply:
This is a concern I also have for a longer time and had a PR open once.

One approach would be to add an alias for every unnamed expression based on the original query SQL or expression.
This would avoid having the column names changed by the optimizers.

Reply:
I really like the idea of adding an alias once (maybe as the initial optimizer pass?)

I am not sure how valuable adding the types in the column names is in general, to be honest. I wouldn't mind if rather than `Int(1)` this was simply rendered `1`

Reply:
> I really like the idea of adding an alias once (maybe as the initial optimizer pass?)
> 
> I am not sure how valuable adding the types in the column names is in general, to be honest. I wouldn't mind if rather than `Int(1)` this was simply rendered `1`

Do you have plan or a draft pr for that? @Dandandan

Reply:
Yes indeed, we can give those a second life 🎉

I had some concerns with the PR, but I believe it is still a big improvement over the current state of things.